lxd/storage/backend_lxd.go

```diff
@@ -2549,6 +2531,51 @@ func (b *lxdBackend) CreateInstanceFromConversion(inst instance.Instance, conn i
 
 	revert.Add(func() { _ = b.driver.DeleteVolume(volCopy.Volume, op) })
 
+	// At this point, the instance's volume is populated. If "virtio" option is enabled,
+	// inject the virtio drivers.
+	if slices.Contains(args.ConversionOptions, "virtio") {
+		b.logger.Debug("Inject virtio drivers started")
+		defer b.logger.Debug("Inject virtio drivers finished")
+
+		err = b.driver.MountVolume(vol, op)
+		if err != nil {
+			return err
+		}
+
+		defer func() { _, _ = b.driver.UnmountVolume(vol, true, op) }()
+
+		diskPath, err := b.driver.GetVolumeDiskPath(vol)
+		if err != nil {
+			return err
+		}
+
+		imgFormat, _, err := b.qemuImageInfo(diskPath)
+		if err != nil {
+			return err
+		}
+
+		// Ensure the disk path points to a disk in raw format.
+		if imgFormat != "raw" {
+			return fmt.Errorf(`Expected volume format for driver injection is "raw", but got %q`, imgFormat)
+		}
+
+		// Run virt-v2v-in-place to inject virtio drivers.
+		cmd := exec.Command(
+			// Use prlimit to limit QEMU to 1 GiB address space and 5 minutes of CPU time.
+			"prlimit", "--cpu=300", "--as=1073741824",
+			"virt-v2v-in-place", "-i", "disk", "-if", "raw", "--block-driver", "virtio-scsi", diskPath,
```

Comment:
We should run this as a low priority, like we do with qemu-img convert using 

"nice", "-n19", // Run with low priority to reduce CPU impact on other processes.

Reply:
For `qemu-img convert` we also use `prlimit`. Should we also run the `qemu-img convert` with low priority instead of limiting it to 5 minutes?

Reply:
both please

Reply:
As discussed lets use prlimit just for the qemu-img info call. Ta